src/import_vdf.py

```diff
@@ -165,6 +293,53 @@ def main():
     parser_qdrant = subparsers.add_parser(DBNames.QDRANT, help="Import data to Qdrant")
     parser_qdrant.add_argument("-u", "--url", type=str, help="Qdrant url")
 
+    # Vertex AI VectorSearch
+    parser_vertexai_vectorsearch = subparsers.add_parser(
+        DBNames.VERTEXAI, help="Import data to Vertex AI Vector Search"
+    )
+    parser_vertexai_vectorsearch.add_argument(
+        "-p", "--project-id", type=str, help="Google Cloud Project ID"
```

Comment:
if not specified, we should provide a list to choose from (based on the logged in user's projects)